libs/pack_traversal/src/pack_traversal.cpp

```diff
@@ -5,30 +5,30 @@
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
Shouldn't this file go into `.../tests/unit`?

Reply:
...

Well done me. Thanks @hkaiser!